pallets/phala/src/compute/vault.rs

```diff
@@ -408,12 +435,7 @@ pub mod pallet {
 
 		/// Demands the return of some stake from a pool.
 		///
-		/// Note: there are two scenarios people may meet
-		///
-		/// - if the pool has free stake and the amount of the free stake is greater than or equal
-		///     to the withdrawal amount (e.g. pool.free_stake >= amount), the withdrawal would
-		///     take effect immediately.
-		/// - else the withdrawal would be queued and delayed until there is enough free stake.
+		/// - The withdrawal would be queued and delayed until there is enough free stake.
```

Comment:
but it would try to process the withdraw on the top of the queue rather than the latest withdraw, so i think some part of  the comment is not needed now

Reply:
done